### DISCUSSION TITLE
Stream not working

### DISCUSSION THREAD

**Discussion by Krispkiwi** [tsightler/ring-mqtt#951] [Support Q&A]
I've enabled the live stream toggle and copied and pasted the rtsp feed from info into VLC under network source and its not pulling the feed. Can anyone help troubleshoot?

**Comment by digimillz** [comment]
I pasted a good amount of my log data from the add on into ChatGPT and told it to build a dashboard that had a snapshot and link to livestream rtsp data that is contained. Worked for me!
